types/wordpress__customize/OverlayNotification.d.ts

```diff
@@ -0,0 +1,6 @@
+import { Notification } from './Notification';
+
+export class OverlayNotification extends Notification {
+	loading: boolean;
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/wordpress__customize/OverlayNotification.d.ts;
  Line: 4;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.